api/providers/serializers.py

```diff
@@ -143,8 +143,12 @@ class Meta:
         'domain_redirect_enabled',
         'id',
         'name',
+        'reviews_workflow',
     ])
 
+    reviews_workflow = ser.ChoiceField(choices=Workflows.choices())
+    reviews_comments_anonymous = ser.BooleanField()
```

Comment:
Should these be `read_only=True`?

Reply:
Yeah, I think so, since this is only intended to be editable via the admin app now?